examples/RESTBase/IntegrationTestExamples.yaml

```diff
@@ -0,0 +1,30 @@
+# Examples are similar to:
+#    * https://github.com/wikimedia/restbase/blob/master/test/features/pagecontent/pagecontent.js#L72
+#    * https://github.com/wikimedia/restbase/blob/master/test/features/pagecontent/pagecontent.js#L56
+#
+# Example run: ./phester.php test https://en.wikipedia.beta.wmflabs.org/api/rest_v1/ examples/RESTBase/IntegrationTestExamples.yaml
+
+suite: RESTBase integration tests
+description: Integration tests
+type: REST
+
+tests:
+  - description: Should transparently create a new HTML revision with id 252937
+    interaction:
+      - request:
+          path: page/html/Foobar/252937
+        response:
+          headers:
+            vary: !pcre/pattern: /^Accept.+/
+  - description: Should transparently create a new HTML revision for Main_Page
+    interaction:
+      - request:
+          path: page/html/Main_Page
+        response:
+          headers:
+            vary: !pcre/pattern: /^Accept.+/
```

Comment:
This is missing the `disallow: ['']` part of the test. It is actually covering a regression that we've had at some point where a 'Vary' header would contain multiple "empty" elements and looked like `Vary: Accept, , , `

Reply:
That boils down to regex wrangling.
Perhaps something like this would work:
/^Accept(,[^,]+)*$/

Reply:
Maybe eventually the need will come to have more sophisticated matchers, as when you have on a problem, just add a regex, and you will have 2 problems :) but for now regexes can cover most of the existing matching we do in restbase tests that I can remember of the top of my head.

Reply:
Just add a regex, you have (problem)+

Reply:
done.